src/components/donation/Steps/Submit/StripeCheckout/Paypal/Paypal.tsx

```diff
@@ -2,23 +2,18 @@ import { PayPalScriptProvider } from "@paypal/react-paypal-js";
 import ContentLoader from "components/ContentLoader";
 import { PAYPAL_CLIENT_ID } from "constants/env";
 import { usePaypalOrderQuery } from "services/apes";
-import { PaypalCheckoutStep, setStep } from "slices/donation";
-import { useSetter } from "store/accessors";
-import { currency } from "../../common/Currrency";
-import Summary from "../../common/Summary";
-import Err from "../Err";
+import { StripeCheckoutStep } from "slices/donation";
 import Checkout from "./Checkout";
 
 // Followed Stripe's custom flow docs
 // https://stripe.com/docs/payments/quickstart
-export default function PaypalCheckout(props: PaypalCheckoutStep) {
+export default function Paypal(props: StripeCheckoutStep) {
   const { details, recipient, liquidSplitPct } = props;
 
   const {
     data: orderId,
     isLoading,
     isError,
-    error,
```

Comment:
Related to this comment about errors, the following error was logged when I updated the code to do so:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/4c10e757-6f97-41a7-8562-f52149c22662)

Reply:
Staging is currently being modified by https://linear.app/better-giving/issue/BG-1164/v2-donations-pipeline-in-apes-aws. Once finished I would  open PR that contains compatible payload types
* https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2719